lib/main.js

```diff
@@ -44,6 +44,10 @@ module.exports = {
     return this.autocompleteManager
   },
 
+  provideManagerAsync () {
```

Comment:
I wanted to emphasize that the service returns a function that takes a callback instead of the actual manager. The `async` suffix might be a bit overspecified, so I changed it to `provideAutocompleteManagerCaller`. Let me know if that still sounds off.